src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -337,8 +354,15 @@ public function setHorizontal(bool $horizontal = true): self
     public function render()
     {
         $divider = new TableSeparator();
-        if ($this->horizontal) {
-            $rows = [];
+        $isCellWithColspan = static function ($cell): bool {
+            return $cell instanceof TableCell && $cell->getColspan() >= 2;
+        };
```

Comment:
Maybe use the short closure syntax?
```suggestion
        $isCellWithColspan = static fn ($cell): bool => $cell instanceof TableCell && $cell->getColspan() >= 2;
```

Reply:
sure